ygen/codegen.go

```diff
@@ -464,7 +465,6 @@ func (cg *YANGCodeGenerator) GenerateGoCode(yangFiles, includePaths []string) (*
 // It returns a GeneratedProto3 struct containing the messages that are to be
 // output, along with any associated values (e.g., enumerations).
 func (cg *YANGCodeGenerator) GenerateProto3(yangFiles, includePaths []string) (*GeneratedProto3, *YANGCodeGeneratorError) {
-	// TODO(github.com/openconfig/ygot/issues/20): Handle enumerated types in proto messages.
 	mdef, errs := mappedDefinitions(yangFiles, includePaths, &cg.Config)
```

Comment:
Why not use one of the already existing error list definitions? e.g. util/errs.go, the one that pictor uses, or https://github.com/hashicorp/go-multierror is a good choice.

Reply:
Yes, I think this was just for consistency, but I've moved everything to util.Errors now. Would prefer to avoid an external dependency.